modules/op-release-notes-1-20-0.adoc

```diff
@@ -0,0 +1,225 @@
+
+* All containers in the `tekton-results` deployment have `readOnlyRootFilesystem` setting enabled, improving security and compliance.
+
+.{pac}
+* {pac} synchronizes the status of running `PipelineRun` on git providers when managed by `Kueue`, a Kubernetes job-queuing system. This enhancement helps developers receive feedback on their pull requests faster through comments and status checks.
+
+* {pac} supports passing incoming webhook parameters in the JSON POST body, while maintaining compatibility with URL query parameters:
++
+[source, terminal]
+----
+curl -H "Content-Type: application/json" -X POST \
+"https://control.pac.url/incoming" -d '{"repository":"repo","branch":"main", \
+"pipelinerun":"target-pipelinerun","secret":"very-secure-shared-secret"}'
+----
++
+Using the request body for sensitive information reduces exposure in logs and improves security.
+
+
+
+* {pac} supports detailed logging for GitHub API calls, providing insights into API interactions, durations, and rate-limiting. By setting the controller's log level to 'debug', you can troubleshoot complex issues more efficiently. This enhancement refactors code to instrument GitHub calls and affects all types of GitHub API calls within the provider.
+
+* {pac} adds a new option in `auto-configure-new-github-repo` feature setting to consolidate repository creation within a single namespace, streamlining management for those with multiple repositories. This helps in creating more organized and manageable environment by autoconfiguring repository names, eliminating the potential chaos caused by multiple namespaces for each repository.
+You can also configure custom repository names with the `auto-configure-repo-repository-template` setting.
++
+[source, yaml]
+----
+#...
+    auto-configure-new-github-repo: "true"
+    auto-configure-repo-namespace-template: "all-repositories-ns"
+    auto-configure-repo-repository-template: "{{repo_owner}}-{{repo_name}}-cr"
+#...
+----
++
+With this example configuration, {pac} creates a `Repository` custom resource (CR) for every new GitHub repository in the `all-repositories-ns` namespace.  
+For example, if you create a repository named `org/test-repo` in GitHub, {pac} creates the corresponding `Repository` CR in the `all-repositories-ns` namespace as `org-test-repo-cr`.
+
+* You can use relative paths to reference tasks from within a remote `Pipeline` definition. The {pac} resolver automatically builds the full URL for the task based on the location of the remote `PipelineSpec` definition. This improves pipeline organization and portability by keeping tasks and pipelines co-located, which is especially useful when working with tagged releases or branches.
+
+.Helm
```

Comment:
we shoudnt have section for helm, it should be included in the component section, for this its on operator